sound/soc/sof/intel/hda-pcm.c

```diff
@@ -118,8 +118,8 @@ int hda_dsp_pcm_hw_params(struct snd_sof_dev *sdev,
 	/* disable SPIB, to enable buffer wrap for stream */
 	hda_dsp_stream_spib_config(sdev, stream, HDA_DSP_SPIB_DISABLE, 0);
 
-	/* set host_period_bytes to 0 if no IPC position */
-	if (hda && hda->no_ipc_position)
+	/* set host_period_bytes to 0 if no irq mode */
+	if (hstream->no_period_wakeup)
```

Comment:
fix the commit message, it mentions a rename which is done in the next patch. I don't understand the part of using the PCM position. this is completely hardware dependent and for HDaudio the position can be provided with the DPIB even outside of interrupts. Please explain what you are trying to do.

Reply:
@plbossart I try to explain my purpose in commit text. this line is...
host_period_bytes is overrided to 0 when the client set nowakeup mode.